pizzabot.js

```diff
@@ -0,0 +1,83 @@
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
+
+document.getElementById(
+  "botBubble"
+).innerHTML = `Hey! Happy to get your order started. On our menu we have ${vegetarian}, ${hawaiian}, and ${pepperoni}.`;
+
+document.getElementById("userBubble").innerHTML =
+  "Ok! (Click to start your order.)";
+
+document.getElementById("userBubble").style.cursor = "pointer";
+
+function buttonClicked() {
+  const buttonInvisible = document.getElementById("button");
+  buttonInvisible.style.display = "none";
+
+  const chatBegin = document.getElementById("chatContainer");
+  chatContainer.style.display = "flex";
+
+  const startConvo = document.getElementById("botBubble");
+  botBubble.style.display = "block";
+
+  const userStart = document.getElementById("userBubble");
+  userStart.style.display = "block";
+}
+
+function startOrder() {
+  const orderForm = document.getElementById("startOrder");
+  orderForm.style.display = "block";
+  document.getElementById(
+    "startOrder"
+  ).innerHTML = `<form class="form" onsubmit="validateOrder(event)">
+ 
+ <div class="pizza_selection">What kind would you like?
+ 
+ <p><input type="radio" id="vegetarian" name="pizza_selection" value="vegetarian" required />
+ <label for="vegetarian">${vegetarian}</label></p>
+ 
+ <p><input type="radio" id="hawaiian" name="pizza_selection" value="hawaiian" required /><label for="hawaiian">${hawaiian}</label></p>
+      
+ <p><input type="radio" id="pepperoni" name="pizza_selection" value="pepperoni" required /><label for="pepperoni">${pepperoni}</label></p></div>
+ 
+ <div class="quantity"><p><label for="orderQty">How many pizzas?</label></p><p><input id="orderQty" type="number" name="orderQty" placeholder="enter a number" required /></p></div>
+ 
+ <p><button class=".orderButton" type="submit" id="submit" name="submit" value="submit">Order</button></p>
+ </form>`;
+}
```

Comment:
I know it's a lot of work, but we shouldn't write code like that, this is completely not readable - try to format this HTML so we can actually see some structure

Reply:
adding in formatting too!